### PR TITLE
BE Update app.py Data generator json object

### PR DESCRIPTION
**Pull request by Jeffchiucp**
- Update app.py
- update the validation for JSON object and snapshot object
-  revise the app.py BE Data generator json object

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -64,39 +64,54 @@ def validate_arrangement(arrangement):
...
-                container_key_id = key
-
-                for item_value_id in value:
-                    if container_key_id == "" or not item_value_id:
-                        break
-
-        if(arrangement_id and name and timestamp and item_id
+                return False
+            else:
+                for key, value in snapshot_dict.items():
```
Comment (deonjo) [inline app.py]: Can you write a helper method to validate snapshot, so it's not so long. and can u get rid of the "if True:"?

File: test/data_generator/data_generator.py [hunk 2]
```diff
@@ -66,63 +66,66 @@
...
-  arrangement.add_item_to_container('jeff', 'chia van')
-  arrangement.add_item_to_container('nathan', 'nathan car')
-  arrangement.add_item_to_container('moses', 'nathan car')
-
-  print(arrangement.build())
-
-if __name__ == "__main__":
-  main()
+
+# def create_random_id(prepended_letter=""):
```
Comment (deonjo) [inline test/data_generator/data_generator.py]: delete

File: test/data_generator/data_generator.py [hunk 3]
```diff
@@ -66,63 +66,66 @@
...
+    data = {}
+
+    def pass_json(self, arrangement):
+        self.data["name"] = arrangement['name']
+        self.data["_id"] = arrangement['_id']
+        self.data["items"] = []
+        self.data["containers"] = []
+        self.data["is_deleted"] = arrangement['is_deleted']
+        self.data["timestamp"] = arrangement['timestamp']
+        self.data["modified_timestamp"] = arrangement['modified_timestamp']
```
Comment (deonjo) [inline test/data_generator/data_generator.py]: thanks for adding it!

File: app.py (hunk 1 above)
Comment (Jeffchiucp) [reply app.py]: done

**Comment by Jeffchiucp** [comment]
update the validate function as @deonjo  requested, let me know if we can merge this

File: test/data_generator/data_generator.py (hunk 3 above)
Comment (Jeffchiucp) [reply test/data_generator/data_generator.py]: np

**Comment by Jeffchiucp** [comment]
@deonjo  appreciate any feedback you have on the backend code refactoring

**Pull request merged**